decosjoin/tests/fixtures/response_mock.py

```diff
@@ -25,7 +27,7 @@ def get_response_mock(self, *args, **kwargs):
         res_data = mocked_get_urls[args[0]]
     except KeyError:
         raise Exception("Url not defined %s", args[0])
-    return MockedResponse(res_data)
+    return MockedResponse(json_=res_data)
```

Comment:
Is dit een json encoder? Ziet er gek uit. Kan het ook niet zo een twee drie hier vinden:
https://docs.python.org/3/library/json.html

Reply:
Nee, dit is een variabele met een trailing underscore. Een patroon gebruikt om aan te geven dat dit een custom interne variabele is, een die niet toebehoort aan de base class